Cronicle Primary LXC

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Cronicle Primary LXC

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/cronicle.sh)"

### 📝 Provide a clear and concise description of the issue.

Ran command and received this outcome

 ⠋bash: line 25: gpg: command not found
 ⠙curl: (23) Failed writing body

[ERROR] in line 25: exit code 0: while executing command gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg

### ⚙️ What settings are you using?

- [ ] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

_No response_

### 🔄 Steps to reproduce the issue.

Step 1 Run bas script to install.

### ❌ Paste the full error output (if available).

Setting up mc (3:4.8.29-2) ...
update-alternatives: using /usr/bin/mcview to provide /usr/bin/view (view) in auto mode
Processing triggers for man-db (2.11.2-2) ...
 ⠙Processing triggers for mailcap (3.70+nmu1) ...
Processing triggers for libc-bin (2.36-9+deb12u10) ...
  ✔️  Installed Dependencies
 ⠋bash: line 25: gpg: command not found
 ⠙curl: (23) Failed writing body

[ERROR] in line 25: exit code 0: while executing command gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg

### 🖼️ Additional context (optional).

_No response_

--- a/install/cronicle-install.sh
+++ b/install/cronicle-install.sh
@@ -17,7 +17,8 @@ msg_info "Installing Dependencies"
 $STD apt-get install -y \
     curl \
     sudo \
-    mc
+    mc \
+    gnupg
 msg_ok "Installed Dependencies"
 
 msg_info "Setting up Node.js Repository"

